backend/routes/notification.py

```
@@ -0,0 +1,34 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from crud.notification import create_notification, get_unread_notifications, get_all_notifications, mark_notification_as_read
```

Ruff (0.8.2)

`crud.notification.create_notification` imported but unused

Remove unused import: `crud.notification.create_notification`

(F401)